diff -crB a/arch/x86/kernel/cpu/bugs.c b/arch/x86/kernel/cpu/bugs.c
*** a/arch/x86/kernel/cpu/bugs.c	2025-02-21 14:11:21.000000000 +0100
--- b/arch/x86/kernel/cpu/bugs.c	2025-02-23 02:28:01.671336723 +0100
***************
*** 1946,1952 ****
  	}
  }
  
- #define MDS_MSG_SMT "MDS CPU bug present and SMT on, data leak possible. See https://www.kernel.org/doc/html/latest/admin-guide/hw-vuln/mds.html for more details.\n"
  #define TAA_MSG_SMT "TAA CPU bug present and SMT on, data leak possible. See https://www.kernel.org/doc/html/latest/admin-guide/hw-vuln/tsx_async_abort.html for more details.\n"
  #define MMIO_MSG_SMT "MMIO Stale Data CPU bug present and SMT on, data leak possible. See https://www.kernel.org/doc/html/latest/admin-guide/hw-vuln/processor_mmio_stale_data.html for more details.\n"
  
--- 1946,1951 ----
***************
*** 1974,1981 ****
  	switch (mds_mitigation) {
  	case MDS_MITIGATION_FULL:
  	case MDS_MITIGATION_VMWERV:
- 		if (sched_smt_active() && !boot_cpu_has(X86_BUG_MSBDS_ONLY))
- 			pr_warn_once(MDS_MSG_SMT);
  		update_mds_branch_idle();
  		break;
  	case MDS_MITIGATION_OFF:
--- 1973,1978 ----
diff -crB a/arch/x86/kernel/cpu/intel_epb.c b/arch/x86/kernel/cpu/intel_epb.c
*** a/arch/x86/kernel/cpu/intel_epb.c	2025-02-21 14:11:21.000000000 +0100
--- b/arch/x86/kernel/cpu/intel_epb.c	2025-02-23 02:28:32.221890782 +0100
***************
*** 108,114 ****
  		val = epb & EPB_MASK;
  		if (val == ENERGY_PERF_BIAS_PERFORMANCE) {
  			val = energ_perf_values[EPB_INDEX_NORMAL];
- 			pr_warn_once("ENERGY_PERF_BIAS: Set to 'normal', was 'performance'\n");
  		}
  	}
  	wrmsrl(MSR_IA32_ENERGY_PERF_BIAS, (epb & ~EPB_MASK) | val);
--- 108,113 ----
diff -crB a/security/lockdown/lockdown.c b/security/lockdown/lockdown.c
*** a/security/lockdown/lockdown.c	2025-02-21 14:11:21.000000000 +0100
--- b/security/lockdown/lockdown.c	2025-02-23 02:31:15.023893916 +0100
***************
*** 30,37 ****
  		return -EPERM;
  
  	kernel_locked_down = level;
- 	pr_notice("Kernel is locked down from %s; see man kernel_lockdown.7\n",
- 		  where);
  	return 0;
  }
  
--- 30,35 ----
***************
*** 63,71 ****
  		return -EPERM;
  
  	if (kernel_locked_down >= what) {
- 		if (lockdown_reasons[what])
- 			pr_notice_ratelimited("Lockdown: %s: %s is restricted; see man kernel_lockdown.7\n",
- 				  current->comm, lockdown_reasons[what]);
  		return -EPERM;
  	}
  
--- 61,66 ----
